sdk/sdk-client/src/utils/PositionUtils.ts

```diff
@@ -0,0 +1,55 @@
+import { Percentage, type Position, type TokenAmount } from '@summerfi/sdk-common/common'
+import BigNumber from 'bignumber.js'
+
+export class PositionUtils {
+  static getLTV({
+    collateralTokenAmount,
+    debtTokenAmount,
+    collateralPrice,
+    debtPrice,
+  }: {
+    collateralTokenAmount: TokenAmount
+    debtTokenAmount: TokenAmount
+    collateralPrice: string
+    debtPrice: string
+  }): Percentage {
+    // Determine the Collateral Value:
+    const collValue = new BigNumber(collateralTokenAmount.amount).times(collateralPrice)
+    // Determine the Borrowed Value:
+    const debtValue = new BigNumber(debtTokenAmount.amount).times(debtPrice)
+    // If the collateral value is 0, return 0.
+    if (collValue.isZero()) {
+      return Percentage.createFrom({ percentage: 0 })
+    }
+    // Divide the borrowed value by the collateral value and multiply by 100 to get the percentage LTV.
+    const ltvRatio = debtValue.div(collValue).times(100).toNumber()
+    return Percentage.createFrom({ percentage: ltvRatio })
+  }
+
+  static getLiquidationPrice({
+    position,
+    liquidationThreshold,
+    debtPrice,
+  }: {
+    position: Position
+    // TODO: passed as param because it is not defined in Position
+    liquidationThreshold: Percentage
```

Comment:
You can get it from Pool, but I think you are not getting Pool from the SDK at this moment. If you don't mind, update the TODO to suggest that we pass Pool in the future

Reply:
done